manifests/base/executionspace/kustomization.yaml

```diff
@@ -5,4 +5,5 @@ resources:
   - rolebinding.yaml
   - service.yaml
   - deployment.yaml
-  - role.yaml
\ No newline at end of file
+  - role.yaml
+  - vars.yaml
```

Comment:
We normally name these files after the resource that is described in them. I.e. `configmap.yaml`, not `vars.yaml`